lib/find-python.js

```diff
@@ -41,7 +41,7 @@ class PythonFinder {
   static findPython = (...args) => new PythonFinder(...args).findPython()
 
   log = log.withPrefix('find Python')
-  argsExecutable = ['-c', 'import sys; print(sys.executable);']
+  argsExecutable = ['-c', 'import sys; sys.stdout.buffer.write(sys.executable.encode(\'utf-8\'));']
```

Comment:
```suggestion
  argsExecutable = ['-c', 'import sys ; sys.stdout.buffer.write(sys.executable.encode("utf-8"));']
```

Reply:
Thank you for your suggestion. I tried it locally but it failed.

Reply:
It failed with what error?!?

Reply:
When I ran the test, I got the following error:
```
1) find-python
       find python - encoding:
     AssertionError [ERR_ASSERTION]: Got unwanted rejection.
Actual message: "The argument 'file' cannot be empty. Received ''"
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async Context.<anonymous> (test\test-find-python.js:61:5)

```